crates/core/tedge_mapper/src/sm_c8y_mapper/mapper.rs

```diff
@@ -88,7 +82,8 @@ impl TEdgeComponent for CumulocitySoftwareManagementMapper {
     #[instrument(skip(self, tedge_config), name = "sm-c8y-mapper")]
     async fn start(&self, tedge_config: TEdgeConfig) -> Result<(), anyhow::Error> {
         let operations = Operations::try_new("/etc/tedge/operations", "c8y")?;
-        let http_proxy = JwtAuthHttpProxy::try_new(&tedge_config).await?;
+        let session_name = "SM-Tedge-Mapper";
```

Comment:
Any specific reason for using a new session name here and not reusing the existing name "SM-C8Y-Mapper" which is already defined in the `SM_MAPPER` const?